views/cellMove/cellMoveViewResidentialLocation.njk

```diff
@@ -0,0 +1,138 @@
+      }
+    ],
+    classes: 'govuk-!-display-none-print'
+  })
+  }}
+{% endblock %}
+
+{% block content %}
+
+  {% if errors.length > 0 %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errors,
+      attributes: { 'data-qa-errors': true }
+      }) 
+    }}
+  {% endif %}
+
+  <h1 class="govuk-heading-l">
+    {{title}}
+  </h1>
+
+  <form novalidate="novalidate" data-test="prisoner-search-form">
+    <div class="horizontal-form govuk-!-margin-bottom-3">
+      {{ govukSelect({
+        id: "location",
+        name: "location",
+        label: {
+          text: "Select residential location"
+        },
+        items: locationOptions | setSelected(formValues.location),
+        attributes: {
+          'data-test': 'prisoner-search-location'
+        }
+      }) }}
+
+      {{ govukButton({
+        text: "View",
+        type: "submit",
+        classes: "govuk-button--secondary"
+      }) }}
+    </div>
+  </form>
+
+  {% if showResults %}
+    <p class="govuk-body pull-right"><strong>Prisoners listed:</strong> {{totalOffenders}}</p>
+
+    {% if results.length %}
+      {% set rows = [] %}
+      {% for prisoner in results %}
+        {% set prisonerImageHtml %}
+          <img src="/app/images/{{prisoner.offenderNo}}/data" alt="Image of {{prisoner.name}}" class="prisoner-search__results-list__image" />
+        {% endset -%}
+        {% set prisonerLinkHtml %}
+          <a href="/prisoner/{{prisoner.offenderNo}}" class="govuk-link">{{prisoner.name}}</a>
+        {% endset -%}
+        {% set cellHistoryLinkHtml %}
+          <a href="{{prisoner.cellHistoryUrl}}" class="govuk-link" data-test="prisoner-cell-history-link">View cell history</a>
+        {% endset -%}
+        {% set cellSearchLinkHtml %}
+          <a href="{{prisoner.cellSearchUrl}}" class="govuk-link" data-test="prisoner-cell-search-link">Change cell</a>
+        {% endset -%}
+        {% set rows = (rows.push([
+          { html: prisonerImageHtml },
+          {
+            html: prisonerLinkHtml,
+            attributes: {
+              "data-sort-value": prisoner.name
+            }
+          },
+          { text: prisoner.offenderNo },
+          { text: prisoner.assignedLivingUnitDesc },
+          { html: alertFlags(prisoner.alerts, newLine=true) + categoryFlag('', prisoner.categoryCode, false) },
+          { html: cellHistoryLinkHtml },
+          { html: cellSearchLinkHtml }
+        ]), rows) %}
+      {% endfor %}
+
+      <div class="prisoner-search__results-list">
+        {{ govukTable({
+          head: [
+            { html: '<span class="govuk-visually-hidden">Picture</span>' },
+            {
+              text: "Name",
+              attributes: {
+                "aria-sort": "ascending"
+              }
+            },
+            { text: "Prison number" },
+            {
+              text: "Location",
+              attributes: {
+                "aria-sort": "ascending"
```

Comment:
I've since realised that only one table header (usually the name one) needs `"aria-sort": "ascending"`.  All the others should be `"aria-sort": "none"`. This will display the correct up/down arrows then but still allow the column to be sortable.

Reply:
Ah ok - makes sense. Fixing